examples/config.yaml

```diff
@@ -0,0 +1,212 @@
+          attribute_keys:
+            - foo
+            - bar
+      # Add a simpler view, which configures the drop aggregation for instruments whose name matches "*.server.duration".
+      - selector:
+          instrument_name: "*.server.duration"
+        view:
+          aggregation:
+            name: drop
+  # Logger provider configuration.
+  logger_provider:
+    resource: resource/resourceA # references identifier from resources section
+    # List of log processors, to be added sequentially. Each log processor has a name and args used to configure it.
+    log_record_processors:
+      # Add batch log processor configured to export with the OTLP exporter.
+      - name: batch
+        args:
+          # Batch log processor takes exporter as an arg
+          exporter: otlp/exporterA # references identifier from exporters section
+          # Configure batch log processor batch size and interval settings.
+          max_queue_size: 50
+          scheduled_delay_millis: 1_000
+          export_timeout_millis: 30_000
+          max_export_batch_size: 200
+  # List of context propagators. Each propagator has a name and args used to configure it. None of the propagators here have configurable options so args are not demonstrated.
+  propagators:
+    - name: tracecontext
+    - name: baggage
+# Using examples provided in https://github.com/MrAlias/otel-schema/issues/8
+instrumentation:
+  java:
+    http:
+      capture_headers:
+        client:
+          request: # otel.instrumentation.http.capture-headers.client.request
+            enabled: true
+            headers:
+              - "User-Agent"
+          response: # otel.instrumentation.http.capture-headers.client.response
+            enabled: false
+        server:
+          request: # otel.instrumentation.http.capture-headers.server.request
+            enabled: false
+          response: # otel.instrumentation.http.capture-headers.server.response
+            enabled: true
+            headers:
+              - "Content-Type"
+  python:
+    falcon:
+      enabled: false # OTEL_PYTHON_DISABLED_INSTRUMENTATIONS
+      excluded_urls: # OTEL_PYTHON_FALCON_EXCLUDED_URLS
+        - http://falcon.com/healthcheck
+    fastapi:
+      excluded_urls: # OTEL_PYTHON_FASTAPI_EXCLUDED_URLS
+        - http://fastapi.com/healthcheck
+    django:
+      request_attributes: # OTEL_PYTHON_DJANGO_TRACED_REQUEST_ATTRS
+        - "content_type"
+        - "path_info"
+# language specific options
+python:
+  context: "contextvars" # OTEL_PYTHON_CONTEXT
+  id_generator:
+    name: "random" # OTEL_PYTHON_ID_GENERATOR
+    # possibly an id generator could require arguments
+  
+  # what would setting these mean for SDK
+  # configuration? would it be parsed by an alternate
+  # SDK?
+  tracer_provider: # OTEL_PYTHON_TRACER_PROVIDER
+  meter_provider: # OTEL_PYTHON_METER_PROVIDER
+  logger_provider: # OTEL_PYTHON_LOGGER_PROVIDER
```

Comment:
Hm... what do these mean?

Reply:
Oh I see... you'd use these to provide an alternative implementation of the SDK?

Configuring an alternative SDK implementation should probably be outside of scope for a file based config scheme.